Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/DemoSpatialAwarenessSceneObjectConsumer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
```

Comment:
Rather than Experimental/SpatialAwareness, could you please use the folder name Experimental/SceneUnderstanding, to more clearly communicate that this feature is about scene understanding? Here and everywhere else. A namespace change to Microsoft.MixedReality.Toolkit.Experimental.SceneUnderstanding would be great also.

Reply:
Great suggestions!

Reply:
Renamed the folder as SceneUnderstanding. Created subfolders per Unity standards, moved files into the folders.